examples/suit_update/README.md

```diff
@@ -1,69 +1,88 @@
 # Overview
 
 This example shows how to integrate SUIT-compliant firmware updates into a
-RIOT application. It implements basic support of the SUIT architecture using the manifest format specified in [draft-moran-suit-manifest-04](https://datatracker.ietf.org/doc/draft-moran-suit-manifest/04/).
+RIOT application. It implements basic support of the SUIT architecture using
+the manifest format specified in
+[draft-moran-suit-manifest-04](https://datatracker.ietf.org/doc/draft-moran-suit-manifest/04/).
+
+Table of contents:
+
+- [Prerequisites][prerequisites]
+- [Setup][setup]
+  - [Sign key generation][key-generation]
+  - [Setup a wired device using ethos][setup-wired]
+    - [Provision the device][setup-wired-provision]
+    - [Configure the network][setup-wired-network]
+  - [Setup a wireless device behind a border router][setup-wireless]
+    - [Provision the wireless device][setup-wireless-provision]
+    - [Configure the wireless network][setup-wireless-network]
+  - [Setup aiocoap fileserver][setup-aiocoap-fileserver]
+- [Perform an update][update]
+  - [Build and publish the firmware update][update-build-publish]
+  - [Notify an update to the device][update-notify]
+- [Detailed explanation][detailed-explanation]
 
 ## Prerequisites
-
-Dependencies:
+[prerequisites]: #Prerequisites
     
-    Python3 : - python > 3.6
-              - ed25519 > 1.4
-              - pyasn1  > 0.4.5
-              - cbor    > 1.0.0 
-              - aiocoap > 0.4
-              - Click   > 7.0
+- Install python dependencies (only Python3.6 and later is supported):
+
+      $ sudo pip install ed25519 pyasn1 cbor click
```

Comment:
I would suggest adding --user. But if you have a reason why this isn't a good idea feel free to ignore, (same for the one below.)

Reply:
The problem with `--user` is that it requires some pip background skills: one has to update its PATH env variable to include `~/.local/bin`. Having it done globally will work for almost all users and we can expect advanced pip users to know how to adapt it to their setup.

Reply:
Ok, thats true. I just don't like installing a non-standard aiocoap version with sudo,